joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/passes/CallLinkerPassTest.scala

```diff
@@ -5,11 +5,33 @@ import io.shiftleft.codepropertygraph.Cpg
 import io.shiftleft.codepropertygraph.generated.nodes.Identifier
 import io.shiftleft.semanticcpg.language._
 
-class CallLinkerPassTest extends DataFlowCodeToCpgSuite {
+class CallLinkerPassTest extends DataFlowCodeToCpgSuite(tsTypes = true) {
 
   "CallLinkerPass" should {
 
     "create call edges correctly for methods from classes" in {
+      val cpg: Cpg = code("""
+          |class Foo {
+          |  a() {}
+          |}
+          |
+          |const f = new Foo();
+          |f.a();
+          |""".stripMargin)
+
+      inside(cpg.identifier("f").l) { case List(constF, fieldF, baseF) =>
+        constF.dynamicTypeHintFullName shouldBe List("Foo")
```

Comment:
This type hint is wrong. It's supposed to be a `fullName` of a `TypeDecl`, not a `name`.